Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ruby:2.3.1
+
+RUN apt-get update -qq
+RUN apt-get install -y build-essential libpq-dev nodejs
+
+WORKDIR /rails-api
+ADD Gemfile /rails-api/
+
+RUN bundle install
```

Comment:
Since this image is intended for development purposes, I would rather not run `bundle install` but let that task to the user whom is making some change to the application.